internal/api/oidc/client.go

```diff
@@ -496,7 +496,16 @@ func (o *OPStorage) userinfoFlows(ctx context.Context, resourceOwner string, use
 			apiFields,
 			action.Script,
 			action.Name,
-			append(actions.ActionToOptions(action), actions.WithHTTP(actionCtx))...,
+			append(actions.ActionToOptions(action), )...,
+		)
+
+		err = actions.Run(
+			actionCtx,
+			ctxFields,
+			apiFields,
+			action.Script,
+			action.Name,
+			append(actions.ActionToOptions(action), actions.WithHTTP(actionCtx), actions.WithUUID(actionCtx))...,
```

Comment:
i'm unsure if this change was intended, but the actions would now be executed twice
please remove the redundancy

Reply:
and i think it would make sense to provide the module in all current action triggers (also the non oidc related ones)

Reply:
alright, don't know how i didn't catch that, this should be solved.